server/src/unifyfs_client_rpc.c

```diff
@@ -911,7 +911,7 @@ static void unifyfs_get_gfids_rpc(hg_handle_t handle)
         /* don't need the input struct any more, so free it now. */
         free(in);
         in = NULL;
-    }
+        }
```

Comment:
fix spacing here

Reply:
> fix spacing here

I Checked the bracket matches the else indent.